src/sentry/south_migrations/0510_alert_rule_triggers.py

```diff
@@ -0,0 +1,1552 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding model 'AlertRuleTriggerExclusion'
+        db.create_table('sentry_alertruletriggerexclusion', (
+            ('id', self.gf('sentry.db.models.fields.bounded.BoundedBigAutoField')(primary_key=True)),
+            ('alert_rule_trigger', self.gf('sentry.db.models.fields.foreignkey.FlexibleForeignKey')(to=orm['sentry.AlertRuleTrigger'])),
+            ('query_subscription', self.gf('sentry.db.models.fields.foreignkey.FlexibleForeignKey')(to=orm['sentry.QuerySubscription'])),
+            ('date_added', self.gf('django.db.models.fields.DateTimeField')(default=datetime.datetime.now)),
+        ))
+        db.send_create_signal('sentry', ['AlertRuleTriggerExclusion'])
+
+        # Adding unique constraint on 'AlertRuleTriggerExclusion', fields ['alert_rule_trigger', 'query_subscription']
+        db.create_unique('sentry_alertruletriggerexclusion', [u'alert_rule_trigger_id', u'query_subscription_id'])
+
+        # Adding model 'AlertRuleTrigger'
+        db.create_table('sentry_alertruletrigger', (
```

Comment:
Interesting that south is adding this table *after* it is referenced by foreign keys in other tables.

Reply:
I noticed that too, seems odd